### ISSUE TITLE
CloudFlare API issue

### ISSUE THREAD

**Issue by mkaand** [skopco/powered-cache#56]
Hi Mustafa,

I changed my CF Global API key today. When I check Powered Cache, If I click Cloudflare on menu it redirects me to Powered Cache CF Settings page. As I remember It used to show submenu if I over on Cloudflare top menu. It seems Powered Cache couldn't update my new API key. It is very strange.

I am make sure my new global API key is working.

**Comment by mustafauysal** [comment]
Hey @mkaand,

I'm confused, CF menu is already a sub-menu under "Powered Cache".

What do you see on `/wp-admin/admin.php?page=powered-cache-extension-cloudflare` page? Doesn't it work?

**Comment by mkaand** [comment]
Hi @mustafauysal 

I used to use Powered Cache for a long time. I was happy with it. But recently I changed my CF API and stopped working even I updated on settings. I know It should be submenu under CF top admin menu but it's gone. If I click CF it goes to CF API Settings page. I updated many times but no luck. Then I switched to Lite Speed Cache. I have two WP website both failed after my API key changed. That's really weird.
